src/infra/runtimeQueryRepo.go

```diff
@@ -15,6 +16,18 @@ import (
 type RuntimeQueryRepo struct {
 }
 
+func (r RuntimeQueryRepo) GetPhpPrimaryConfFilePath(
+	hostname valueObject.Fqdn,
+) string {
+	primaryConfFile := "/app/conf/php/primary.conf"
+	mainVirtualHost := valueObject.NewFqdnPanic(os.Getenv("VIRTUAL_HOST"))
+	if hostname != mainVirtualHost {
+		primaryConfFile = "/app/domains/" + string(hostname) + "/conf/php/primary.conf"
+	}
+
+	return primaryConfFile
+}
+
```

Comment:
1) Nunca usamos Panic em infra.
2) Existe o helper "getPrimaryHostname", portanto NUNCA devemos usar o os.Getenv(VIRTUAL_HOST) em nenhum outro lugar exceto no helper em si:
https://github.com/speedianet/os/blob/main/src/infra/helper/getPrimaryHostname.go
3) O arquivo que config primário SEMPRE será o "/app/conf/php/primary.conf". Se o objetivo do seu método era pegar o ConfPath da config do PHP de um vhost, então ele deve se chamar "GetVirtualHostPhpConfFilePath" para deixar claro essa intenção.
4) Nunca usamos string(vo), sempre vo.String().
5) Na versão anterior do SpeediaOS, sempre que adicionávamos um vhost, era criado um vhost no OLS pois ele era o webserver principal. Agora isso não acontece mais, pois o webserver principal é o NGINX. Minha pergunta é: como você teve sucesso no teste unitário do PHP com um sub-domínio se o .conf do sub-domínio nunca tá sendo criado? Se eu entendi a lógica bem, tá faltando todo um fluxo lá no MappingCmdRepo para que quando um novo mapping for criado com o destino para o PHP, que seja criado o .conf do hostname desse mapping.

Reply:
1. Ajustado.
2. Ajustado.
3. Ajustado.
4. Ajustado.
5. Ajustando...